src/routes/user.js

```diff
@@ -0,0 +1,65 @@
+const express = require('express')
+const router = new express.Router()
+const User = require('../models/user')
+const mongoose = require('mongoose')
+
+router.post('/users', async(req, res) => {
+    const user = new User(req.body)
+        try{
+            await user.save()
```

Comment:
You should move the code that calls APIs/DB into a services folder. Create services/user.js and move the DB code to a functions there and call the functions inside the controller which is this file.